src/core/Akka/Helios.Concurrency.DedicatedThreadPool.cs

```diff
@@ -299,7 +312,7 @@ public DedicatedThreadPool(DedicatedThreadPoolSettings settings)
         /// This exception is thrown if the given <paramref name="work"/> item is undefined.
         /// </exception>
         /// <returns>TBD</returns>
-        public bool QueueUserWorkItem(Action work)
+        public bool QueueUserWorkItem(IRunnable work)
```

Comment:
Changed all signatures on `DedicatedThreadPool` from `Action` to `IRunnable` - so we can just call the `IRunnable.Run()` method directly.